fearless/Modules/Staking/SelectValidatorsFlow/ValidatorInfo/ValidatorInfoViewController.swift

```diff
@@ -44,6 +47,29 @@ final class ValidatorInfoViewController: UIViewController, ViewHolder {
             .stakingValidatorInfoTitle(preferredLanguages: selectedLocale.rLanguages)
     }
 
+    func applyState() {
+        guard let state = state else {
+            return
+        }
+
+        didStopLoading()
```

Comment:
`view?.didRecieve(state: .loading)` and here loading stop immediately

Reply:
Yes, and starts loading again if state is loading. Do you think this is a problem?